internal/kafka/internal/config/kafka.go

```diff
@@ -65,13 +68,33 @@ func (c *KafkaConfig) ReadFiles() error {
 	if err != nil {
 		return err
 	}
-	content, err := shared.ReadFile(c.KafkaCapacityConfigFile)
+
+	SupportedInstanceTypesFileContent, err := shared.ReadFile(c.SupportedInstanceTypes.ConfigurationFile)
```

Comment:
there is a new method added to shared lib that can read yaml files directly, https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/75ee96dc51150d2ae51a4a8eecc8ae036b1e2135/pkg/shared/config.go#L84 better to use that

Reply:
:+1:  I added a new commit with the change